src/compas_timber/design/wall_from_surface.py

```diff
@@ -359,20 +363,27 @@ def offset_elements(self, element_loop):
                 element.offset(self.edge_stud_offset)
             offset_loop.append(element)
             # self.edges.append(Line(element.centerline[0], element.centerline[1]))
+
         for i, element in enumerate(offset_loop):
             if self.edge_stud_offset > 0:
                 if element.type != "plate":
                     element_before = offset_loop[i - 1]
                     element_after = offset_loop[(i + 1) % len(offset_loop)]
-                    start_point = intersection_line_line(element.centerline, element_before.centerline, 0.01)[0]
-                    end_point = intersection_line_line(element.centerline, element_after.centerline, 0.01)[0]
+                    start_point = intersection_line_line(
```

Comment:
iirc one or both points can come back as `None` maybe check for that and raise some meaningful error so user knows what they did wrong